helm/presets/aws.yaml

```diff
@@ -36,9 +36,23 @@ strimzi:
       limits:
         cpu: 250m
         memory: 512Mi
+ingress:
+  hosts:
+    - host: hindsight.com
```

Comment:
We should consider setting this to `""` and requiring it via a Go macro. We're doing this elsewhere.

Reply:
the `required` function in the templates?
i don't really like any of the ways to handle this regardless since it's handled as a list and to override any part of it you basically have to override the whole thing. If we want to be more authoritative in how people set up their ingress rules this gets easier, but right now you can arbitrarily set any number of virtual hostnames to respond to an arbitrary number of paths which makes setting up an open-ended values file with sane defaults tricky.

I can save on the complexity by having a single hostname respond to one path each for acquire, broadcast, and eventually receive, too. But someone else may want to have a different virtual host handle access to each service with multiple paths or no paths each (each service just goes to that host's `"/"`) and that's totally valid too.

Thoughts?

Reply:
I think it's completely fine if we want to be opinionated up front about how ingress should be set up. Anyone who wants something else can download the chart, make their change, and use that change.

So my vote is for setting the single hostname (that is configurable and required).

Reply:
Addresses; assumes a single host for the moment.